Fix note

ali-ince opened the pull request:
The PR removes a note which is no more correct since drivers 1.7 and server 3.5. Not sure whether we should highlight that the new behaviour is only valid when using 1.7 driver against a 3.5+ server?

ali-ince committed:
Remove the note which isn't correct for 1.7+ and 3.5+

mariascharin commented on asciidoc/sessions-and-transactions.adoc:
```diff
@@ -263,16 +263,10 @@ When working with a Causal Cluster, transactions can be chained to ensure causal
-Due to protocol limitations, <<driver-transactions-auto-commit, auto-commit transactions>> cannot currently take part in the causal chain.
-While this is expected to change in a future protocol release, `session.run` calls should be avoided in places where causal consistency is important.
---
-
```
@ali-ince Thank you!
Since there may be configurations that 1.7+ driver with pre-3.5 server I think we need to keep a note in there.
It could say something like:

[NOTE]	
--	
If you use the {driver-version} driver to connect to a Neo4j database that is version 3.4 or earlier, <<driver-transactions-auto-commit, auto-commit transactions>> cannot take part in the causal chain.	
In that configuration, `session.run` calls should be avoided in places where causal consistency is important.	
--	

It could possibly be moved to a less prominent place at the end of the section.

ali-ince replied:
Thanks @mariascharin. I've added the note just following the previous paragraph as I thought will be more visible.

ali-ince committed:
Add note regarding causal chaining with auto-commit transactions

With 1.7 driver against a server version 3.4 or less, causal chaining won't work with auto-commit transactions.
